### ISSUE TITLE
Multiple PRs connected to one issue, moves all PRs

### ISSUE THREAD

**Issue by stevenmccord** [ZenHubIO/support#1145]
My scenario is that I have an issue with two pull requests associated with it.  I move one PR to a pipeline and it moves the other PR and issue to that pipeline.  I believe it used to allow that the PRs linked to a ticket could be in another pipeline.  This was nice b/c one PR could be ahead of another and we would deploy that before the other while the other PR is in progress.  Not sure if this was a feature enhancement or currently a bug.  Thanks!

**Comment by kendramoroz** [comment]
@stevenmccord Thanks for getting in touch! Using the Issue to PR connection feature will create one unit of work, as you mentioned! This means all other connected PRs and the issue itself will move seamlessly as the same unit of work. If you want to move one PR separately from the others, but still have it reference, instead of using the first-class supported issue to PR linking feature, have you given our latest dependency feature a spin? This will create a visual indication that the Issue is blocked by a PR, but not have them unified as one unit of work on the board :). 

![image](https://user-images.githubusercontent.com/11036230/26898666-9d1b2d14-4b81-11e7-985f-482c5e87f7c8.png)

Let me know if this helps answer the question or if I can help with anything else!

**Comment by stevenmccord** [comment]
Yeah, I utilize that too, but I guess you guys see the PRs as moving with the issue, I liked it being able to move separate from the issue and other pull requests, but I can see what you are doing there.  I guess we wouldn't really link more than one PR if they aren't going to be deployed together.
